rdiaz593/bbva_test
# Adding black and flake8 to GitHub Actions

### Description:
This PR introduces **GitHub Actions** configuration to run **Black** and **Flake8** on every push or pull request to the `main` branch. The integration ensures that code is properly formatted and follows PEP 8 style rules.

### Major changes:
- Added `.github/workflows/python-lint.yml` file to configure GitHub Actions.
- **Flake8** checks for style errors and potential issues in code.
- **Black** checks code formatting using the `--check` option to ensure style consistency.

### How to test:
- Run `flake8 .` and `black --check .` locally to check formatting and style before pushing.
